lerna.json

```diff
@@ -1,5 +1,5 @@
 {
-  "lerna": "2.0.0-beta.38",
+  "lerna": "2.0.0-beta.23",
```

Comment:
Just wanted to note this change is unrelated to this PR. Without this, I get this when trying to `lerna bootstrap`: